--- case
+++ expected
@@ -56,8 +56,8 @@
 }
 
 GRAPH sensed:zz_moisture {
-  orexis:obs_zz_moisture a orexis:band.zz.moisture.inside .
-  orexis:obs_zz_moisture a sensing:InRegion .
+  orexis:obs_zz_moisture a orexis:band.zz.moisture.below .
+  orexis:obs_zz_moisture a sensing:BelowRegion .
   orexis:obs_zz_moisture a sosa:Observation .
   orexis:obs_zz_moisture sosa:hasFeatureOfInterest :zz .
   orexis:obs_zz_moisture sosa:observedProperty :moisture .
@@ -65,9 +65,9 @@
 
 GRAPH result:zz_moisture {
   orexis:obs_zz_moisture prov:wasGeneratedBy :keeper .
-  orexis:obs_zz_moisture sosa:hasSimpleResult 0.22 .
+  orexis:obs_zz_moisture sosa:hasSimpleResult 0.08 .
   orexis:obs_zz_moisture sosa:madeBySensor :probe .
-  orexis:obs_zz_moisture sosa:resultTime "2026-01-01T11:00:00+00:00"^^xsd:dateTime .
+  orexis:obs_zz_moisture sosa:resultTime "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
 }
 
 GRAPH :catalogue {
@@ -80,13 +80,13 @@
   :world a orexis:PublicGraph .
   result:zz_moisture a orexis:BeliefGraph .
   result:zz_moisture a sensing:ResultGraph .
-  result:zz_moisture dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T11:15:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T11:00:00+00:00"^^xsd:dateTime ] .
+  result:zz_moisture dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:15:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   result:zz_moisture orexis:arrivedBy orexis:Received .
   result:zz_moisture orexis:beliefsOf :keeper .
   sensed:zz_moisture a orexis:BeliefGraph .
   sensed:zz_moisture a orexis:Graph .
   sensed:zz_moisture a orexis:StateGraph .
-  sensed:zz_moisture dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T11:15:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T11:00:00+00:00"^^xsd:dateTime ] .
+  sensed:zz_moisture dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:15:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   sensed:zz_moisture orexis:arrivedBy orexis:Received .
   sensed:zz_moisture orexis:beliefsOf :keeper .
 }
