--- case
+++ expected
@@ -88,4 +88,334 @@
   :sensed a orexis:StateGraph .
   :world a orexis:Graph .
   :world a orexis:PublicGraph .
-}
+  ground:2026-01-01T120000p0000 a planning:GroundGraph .
+  ground:2026-01-01T120000p0000 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  ground:2026-01-01T120000p0000 orexis:arrivedBy orexis:Derived .
+  ground:2026-01-01T120000p0000 orexis:hash "sha256:3da550989c3e7a9c5a0a850a82b91a2ab3e8f2878f27eeb7631e00672b1b0665" .
+  ground:2026-01-01T120000p0000.for.keeper.in_range a planning:Weighing .
+  ground:2026-01-01T120000p0000.for.keeper.in_range planning:for :keeper.in_range .
+  ground:2026-01-01T120000p0000.for.keeper.in_range planning:met false .
+  ground:2026-01-01T120000p0000.for.keeper.in_range planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ; planning:offending 0 ] .
+  ground:2026-01-01T120000p0000.for.keeper.in_range planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank2 ; planning:offending 0 ] .
+  ground:2026-01-01T120000p0000.for.keeper.in_range planning:weighs ground:2026-01-01T120000p0000 .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:expanded true .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:met false .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ; planning:offending 0 ] .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:weighs ground:2026-01-01T120000p0000 .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 a planning:Weighing .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:expanded true .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:met false .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank2 ; planning:offending 0 ] .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:weighs ground:2026-01-01T120000p0000 .
+  plan:keeper.in_range.pursued.tank1 a planning:PlanGraph .
+  plan:keeper.in_range.pursued.tank1 orexis:arrivedBy orexis:Derived .
+  plan:keeper.in_range.pursued.tank1.1.adds a execution:AddsGraph .
+  plan:keeper.in_range.pursued.tank1.1.adds orexis:arrivedBy orexis:Derived .
+  plan:keeper.in_range.pursued.tank1.1.retracts a execution:RetractsGraph .
+  plan:keeper.in_range.pursued.tank1.1.retracts orexis:arrivedBy orexis:Derived .
+  plan:keeper.in_range.pursued.tank1.3.adds a execution:AddsGraph .
+  plan:keeper.in_range.pursued.tank1.3.adds orexis:arrivedBy orexis:Derived .
+  plan:keeper.in_range.pursued.tank1.3.retracts a execution:RetractsGraph .
+  plan:keeper.in_range.pursued.tank1.3.retracts orexis:arrivedBy orexis:Derived .
+  plan:keeper.in_range.pursued.tank2 a planning:PlanGraph .
+  plan:keeper.in_range.pursued.tank2 orexis:arrivedBy orexis:Derived .
+  plan:keeper.in_range.pursued.tank2.2.adds a execution:AddsGraph .
+  plan:keeper.in_range.pursued.tank2.2.adds orexis:arrivedBy orexis:Derived .
+  plan:keeper.in_range.pursued.tank2.2.retracts a execution:RetractsGraph .
+  plan:keeper.in_range.pursued.tank2.2.retracts orexis:arrivedBy orexis:Derived .
+  plan:keeper.in_range.pursued.tank2.6.adds a execution:AddsGraph .
+  plan:keeper.in_range.pursued.tank2.6.adds orexis:arrivedBy orexis:Derived .
+  plan:keeper.in_range.pursued.tank2.6.retracts a execution:RetractsGraph .
+  plan:keeper.in_range.pursued.tank2.6.retracts orexis:arrivedBy orexis:Derived .
+  possible:1 a planning:PossibleGraph .
+  possible:1 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  possible:1 orexis:arrivedBy orexis:Derived .
+  possible:1 orexis:hash "sha256:c80ff1f6e0ffce1c8575ebd35a6183b376384efef6139df68098f31e3b94921e" .
+  possible:1 planning:by possible:1.by .
+  possible:1 planning:minted 1 .
+  possible:1 planning:spent 1.0 .
+  possible:1.by :tank :tank1 .
+  possible:1.by a planning:Candidate .
+  possible:1.by planning:fills :fill .
+  possible:1.by planning:from ground:2026-01-01T120000p0000 .
+  possible:1.by planning:minted 1 .
+  possible:1.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  possible:1.for.keeper.in_range.pursued.tank1 planning:expanded true .
+  possible:1.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  possible:1.for.keeper.in_range.pursued.tank1 planning:met false .
+  possible:1.for.keeper.in_range.pursued.tank1 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ; planning:offending 5 ] .
+  possible:1.for.keeper.in_range.pursued.tank1 planning:weighs possible:1 .
+  possible:1.for.keeper.in_range.pursued.tank2 a planning:Weighing .
+  possible:1.for.keeper.in_range.pursued.tank2 planning:expanded true .
+  possible:1.for.keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
+  possible:1.for.keeper.in_range.pursued.tank2 planning:met false .
+  possible:1.for.keeper.in_range.pursued.tank2 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank2 ; planning:offending 0 ] .
+  possible:1.for.keeper.in_range.pursued.tank2 planning:weighs possible:1 .
+  possible:2 a planning:PossibleGraph .
+  possible:2 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  possible:2 orexis:arrivedBy orexis:Derived .
+  possible:2 orexis:hash "sha256:b925a49cc433542befaf03149f9d7f1e92e07473431966218677da6fae49dc78" .
+  possible:2 planning:by possible:2.by .
+  possible:2 planning:minted 2 .
+  possible:2 planning:spent 1.0 .
+  possible:2.by :tank :tank2 .
+  possible:2.by a planning:Candidate .
+  possible:2.by planning:fills :fill .
+  possible:2.by planning:from ground:2026-01-01T120000p0000 .
+  possible:2.by planning:minted 2 .
+  possible:2.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  possible:2.for.keeper.in_range.pursued.tank1 planning:expanded true .
+  possible:2.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  possible:2.for.keeper.in_range.pursued.tank1 planning:met false .
+  possible:2.for.keeper.in_range.pursued.tank1 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ; planning:offending 0 ] .
+  possible:2.for.keeper.in_range.pursued.tank1 planning:weighs possible:2 .
+  possible:2.for.keeper.in_range.pursued.tank2 a planning:Weighing .
+  possible:2.for.keeper.in_range.pursued.tank2 planning:expanded true .
+  possible:2.for.keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
+  possible:2.for.keeper.in_range.pursued.tank2 planning:met false .
+  possible:2.for.keeper.in_range.pursued.tank2 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank2 ; planning:offending 5 ] .
+  possible:2.for.keeper.in_range.pursued.tank2 planning:weighs possible:2 .
+  possible:3 a planning:PossibleGraph .
+  possible:3 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  possible:3 orexis:arrivedBy orexis:Derived .
+  possible:3 orexis:hash "sha256:262245b8ab6803ce947b3d056e7289f4f5b6c0c8a36cd94f06c2cb130be5235f" .
+  possible:3 planning:by possible:3.by .
+  possible:3 planning:minted 3 .
+  possible:3 planning:spent 2.0 .
+  possible:3.by :tank :tank1 .
+  possible:3.by a planning:Candidate .
+  possible:3.by planning:fills :fill .
+  possible:3.by planning:from possible:1 .
+  possible:3.by planning:minted 3 .
+  possible:3.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  possible:3.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  possible:3.for.keeper.in_range.pursued.tank1 planning:met true .
+  possible:3.for.keeper.in_range.pursued.tank1 planning:weighs possible:3 .
+  possible:3.for.keeper.in_range.pursued.tank2 a planning:Weighing .
+  possible:3.for.keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
+  possible:3.for.keeper.in_range.pursued.tank2 planning:met false .
+  possible:3.for.keeper.in_range.pursued.tank2 planning:open true .
+  possible:3.for.keeper.in_range.pursued.tank2 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank2 ; planning:offending 0 ] .
+  possible:3.for.keeper.in_range.pursued.tank2 planning:weighs possible:3 .
+  possible:4 a planning:PossibleGraph .
+  possible:4 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  possible:4 orexis:arrivedBy orexis:Derived .
+  possible:4 orexis:hash "sha256:886db30e45d2ce9df71bfa463e64da2ed0b025d20c419774efde962241fa9582" .
+  possible:4 planning:by possible:4.by .
+  possible:4 planning:minted 4 .
+  possible:4 planning:spent 2.0 .
+  possible:4.by :tank :tank2 .
+  possible:4.by a planning:Candidate .
+  possible:4.by planning:fills :fill .
+  possible:4.by planning:from possible:1 .
+  possible:4.by planning:minted 4 .
+  possible:4.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  possible:4.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  possible:4.for.keeper.in_range.pursued.tank1 planning:met false .
+  possible:4.for.keeper.in_range.pursued.tank1 planning:open true .
+  possible:4.for.keeper.in_range.pursued.tank1 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ; planning:offending 5 ] .
+  possible:4.for.keeper.in_range.pursued.tank1 planning:weighs possible:4 .
+  possible:4.for.keeper.in_range.pursued.tank2 a planning:Weighing .
+  possible:4.for.keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
+  possible:4.for.keeper.in_range.pursued.tank2 planning:met false .
+  possible:4.for.keeper.in_range.pursued.tank2 planning:open true .
+  possible:4.for.keeper.in_range.pursued.tank2 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank2 ; planning:offending 5 ] .
+  possible:4.for.keeper.in_range.pursued.tank2 planning:weighs possible:4 .
+  possible:5 a planning:PossibleGraph .
+  possible:5 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  possible:5 orexis:arrivedBy orexis:Derived .
+  possible:5 orexis:hash "sha256:886db30e45d2ce9df71bfa463e64da2ed0b025d20c419774efde962241fa9582" .
+  possible:5 planning:by possible:5.by .
+  possible:5 planning:minted 5 .
+  possible:5 planning:spent 2.0 .
+  possible:5.by :tank :tank1 .
+  possible:5.by a planning:Candidate .
+  possible:5.by planning:fills :fill .
+  possible:5.by planning:from possible:2 .
+  possible:5.by planning:minted 5 .
+  possible:5.by.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  possible:5.by.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  possible:5.by.for.keeper.in_range.pursued.tank1 planning:repeats possible:4 .
+  possible:5.by.for.keeper.in_range.pursued.tank1 planning:weighs possible:5.by .
+  possible:5.by.for.keeper.in_range.pursued.tank2 a planning:Weighing .
+  possible:5.by.for.keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
+  possible:5.by.for.keeper.in_range.pursued.tank2 planning:repeats possible:4 .
+  possible:5.by.for.keeper.in_range.pursued.tank2 planning:weighs possible:5.by .
+  possible:6 a planning:PossibleGraph .
+  possible:6 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  possible:6 orexis:arrivedBy orexis:Derived .
+  possible:6 orexis:hash "sha256:71af2c683666133f094033222ad82f45676a17913532f4ca8f587ecc30cc9993" .
+  possible:6 planning:by possible:6.by .
+  possible:6 planning:minted 6 .
+  possible:6 planning:spent 2.0 .
+  possible:6.by :tank :tank2 .
+  possible:6.by a planning:Candidate .
+  possible:6.by planning:fills :fill .
+  possible:6.by planning:from possible:2 .
+  possible:6.by planning:minted 6 .
+  possible:6.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  possible:6.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  possible:6.for.keeper.in_range.pursued.tank1 planning:met false .
+  possible:6.for.keeper.in_range.pursued.tank1 planning:open true .
+  possible:6.for.keeper.in_range.pursued.tank1 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ; planning:offending 0 ] .
+  possible:6.for.keeper.in_range.pursued.tank1 planning:weighs possible:6 .
+  possible:6.for.keeper.in_range.pursued.tank2 a planning:Weighing .
+  possible:6.for.keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
+  possible:6.for.keeper.in_range.pursued.tank2 planning:met true .
+  possible:6.for.keeper.in_range.pursued.tank2 planning:weighs possible:6 .
+  pursued:2026-01-01T120000p0000--open a orexis:Graph .
+  pursued:2026-01-01T120000p0000--open a planning:WantGraph .
+  pursued:2026-01-01T120000p0000--open dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  pursued:2026-01-01T120000p0000--open orexis:arrivedBy orexis:Derived .
+  pursued:2026-01-01T120000p0000--open orexis:beliefsOf :keeper .
+}
+
+
+GRAPH ground:2026-01-01T120000p0000 {
+  :tank1 :level 0 .
+  :tank2 :level 0 .
+}
+
+GRAPH plan:keeper.in_range.pursued.tank1 {
+  plan:keeper.in_range.pursued.tank1 a planning:Plan .
+  plan:keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  plan:keeper.in_range.pursued.tank1 planning:outcome planning:Satisfied .
+  plan:keeper.in_range.pursued.tank1 planning:spent 2.0 .
+  plan:keeper.in_range.pursued.tank1.1 :tank :tank1 .
+  plan:keeper.in_range.pursued.tank1.1 a execution:Step .
+  plan:keeper.in_range.pursued.tank1.1 execution:adds plan:keeper.in_range.pursued.tank1.1.adds .
+  plan:keeper.in_range.pursued.tank1.1 execution:landsAt "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank1.1 execution:notAfter "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank1.1 execution:notBefore "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank1.1 execution:partOf plan:keeper.in_range.pursued.tank1 .
+  plan:keeper.in_range.pursued.tank1.1 execution:retracts plan:keeper.in_range.pursued.tank1.1.retracts .
+  plan:keeper.in_range.pursued.tank1.1 execution:then plan:keeper.in_range.pursued.tank1.3 .
+  plan:keeper.in_range.pursued.tank1.1 planning:fills :fill .
+  plan:keeper.in_range.pursued.tank1.1 planning:of possible:1.by .
+  plan:keeper.in_range.pursued.tank1.3 :tank :tank1 .
+  plan:keeper.in_range.pursued.tank1.3 a execution:Step .
+  plan:keeper.in_range.pursued.tank1.3 execution:adds plan:keeper.in_range.pursued.tank1.3.adds .
+  plan:keeper.in_range.pursued.tank1.3 execution:landsAt "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank1.3 execution:notAfter "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank1.3 execution:notBefore "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank1.3 execution:partOf plan:keeper.in_range.pursued.tank1 .
+  plan:keeper.in_range.pursued.tank1.3 execution:retracts plan:keeper.in_range.pursued.tank1.3.retracts .
+  plan:keeper.in_range.pursued.tank1.3 planning:fills :fill .
+  plan:keeper.in_range.pursued.tank1.3 planning:of possible:3.by .
+}
+
+GRAPH plan:keeper.in_range.pursued.tank1.1.adds {
+  :tank1 :level 5 .
+}
+
+GRAPH plan:keeper.in_range.pursued.tank1.1.retracts {
+  :tank1 :level 0 .
+}
+
+GRAPH plan:keeper.in_range.pursued.tank1.3.adds {
+  :tank1 :level 10 .
+}
+
+GRAPH plan:keeper.in_range.pursued.tank1.3.retracts {
+  :tank1 :level 5 .
+}
+
+GRAPH plan:keeper.in_range.pursued.tank2 {
+  plan:keeper.in_range.pursued.tank2 a planning:Plan .
+  plan:keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
+  plan:keeper.in_range.pursued.tank2 planning:outcome planning:Satisfied .
+  plan:keeper.in_range.pursued.tank2 planning:spent 2.0 .
+  plan:keeper.in_range.pursued.tank2.2 :tank :tank2 .
+  plan:keeper.in_range.pursued.tank2.2 a execution:Step .
+  plan:keeper.in_range.pursued.tank2.2 execution:adds plan:keeper.in_range.pursued.tank2.2.adds .
+  plan:keeper.in_range.pursued.tank2.2 execution:landsAt "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank2.2 execution:notAfter "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank2.2 execution:notBefore "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank2.2 execution:partOf plan:keeper.in_range.pursued.tank2 .
+  plan:keeper.in_range.pursued.tank2.2 execution:retracts plan:keeper.in_range.pursued.tank2.2.retracts .
+  plan:keeper.in_range.pursued.tank2.2 execution:then plan:keeper.in_range.pursued.tank2.6 .
+  plan:keeper.in_range.pursued.tank2.2 planning:fills :fill .
+  plan:keeper.in_range.pursued.tank2.2 planning:of possible:2.by .
+  plan:keeper.in_range.pursued.tank2.6 :tank :tank2 .
+  plan:keeper.in_range.pursued.tank2.6 a execution:Step .
+  plan:keeper.in_range.pursued.tank2.6 execution:adds plan:keeper.in_range.pursued.tank2.6.adds .
+  plan:keeper.in_range.pursued.tank2.6 execution:landsAt "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank2.6 execution:notAfter "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank2.6 execution:notBefore "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  plan:keeper.in_range.pursued.tank2.6 execution:partOf plan:keeper.in_range.pursued.tank2 .
+  plan:keeper.in_range.pursued.tank2.6 execution:retracts plan:keeper.in_range.pursued.tank2.6.retracts .
+  plan:keeper.in_range.pursued.tank2.6 planning:fills :fill .
+  plan:keeper.in_range.pursued.tank2.6 planning:of possible:6.by .
+}
+
+GRAPH plan:keeper.in_range.pursued.tank2.2.adds {
+  :tank2 :level 5 .
+}
+
+GRAPH plan:keeper.in_range.pursued.tank2.2.retracts {
+  :tank2 :level 0 .
+}
+
+GRAPH plan:keeper.in_range.pursued.tank2.6.adds {
+  :tank2 :level 10 .
+}
+
+GRAPH plan:keeper.in_range.pursued.tank2.6.retracts {
+  :tank2 :level 5 .
+}
+
+GRAPH possible:1 {
+  :tank1 :level 5 .
+  :tank2 :level 0 .
+}
+
+GRAPH possible:2 {
+  :tank1 :level 0 .
+  :tank2 :level 5 .
+}
+
+GRAPH possible:3 {
+  :tank1 :level 10 .
+  :tank2 :level 0 .
+}
+
+GRAPH possible:4 {
+  :tank1 :level 5 .
+  :tank2 :level 5 .
+}
+
+GRAPH possible:5 {
+  :tank1 :level 5 .
+  :tank2 :level 5 .
+}
+
+GRAPH possible:6 {
+  :tank1 :level 0 .
+  :tank2 :level 10 .
+}
+
+GRAPH pursued:2026-01-01T120000p0000--open {
+  :keeper planning:holds :keeper.in_range.pursued.tank1 .
+  :keeper planning:holds :keeper.in_range.pursued.tank2 .
+  :keeper.in_range.pursued.tank1 a planning:Want .
+  :keeper.in_range.pursued.tank1 planning:metWhen :keeper.in_range.pursued.tank1.met .
+  :keeper.in_range.pursued.tank1 planning:state planning:Recognized .
+  :keeper.in_range.pursued.tank1 prov:generatedAtTime "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  :keeper.in_range.pursued.tank1 prov:wasDerivedFrom :keeper.in_range .
+  :keeper.in_range.pursued.tank1 rdfs:label "pursued: every tank in range" .
+  :keeper.in_range.pursued.tank1.met a sh:NodeShape .
+  :keeper.in_range.pursued.tank1.met sh:property [ planning:about :level ; sh:minInclusive 10 ; sh:path :level ] .
+  :keeper.in_range.pursued.tank1.met sh:targetNode :tank1 .
+  :keeper.in_range.pursued.tank2 a planning:Want .
+  :keeper.in_range.pursued.tank2 planning:metWhen :keeper.in_range.pursued.tank2.met .
+  :keeper.in_range.pursued.tank2 planning:state planning:Recognized .
+  :keeper.in_range.pursued.tank2 prov:generatedAtTime "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
+  :keeper.in_range.pursued.tank2 prov:wasDerivedFrom :keeper.in_range .
+  :keeper.in_range.pursued.tank2 rdfs:label "pursued: every tank in range" .
+  :keeper.in_range.pursued.tank2.met a sh:NodeShape .
+  :keeper.in_range.pursued.tank2.met sh:property [ planning:about :level ; sh:minInclusive 10 ; sh:path :level ] .
+  :keeper.in_range.pursued.tank2.met sh:targetNode :tank2 .
+}
