--- case
+++ expected
@@ -78,7 +78,17 @@
   :ontology a orexis:PublicGraph .
   :world a orexis:Graph .
   :world a orexis:PublicGraph .
+  <http://example.org/orexis/graph/ground/2026-01-01T120000p0000> a <http://example.org/orexis/planning#GroundGraph> .
+  <http://example.org/orexis/graph/ground/2026-01-01T120000p0000> dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:15:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  <http://example.org/orexis/graph/ground/2026-01-01T120000p0000> orexis:arrivedBy orexis:Derived .
+  <http://example.org/orexis/graph/ground/2026-01-01T120000p0000> orexis:hash "sha256:cbed4defc605dc72abebccd2e9ee207ab076873cbb0556ae85419dfe80ec3149" .
+  <http://example.org/orexis/graph/ground/2026-01-01T121500p0000> a <http://example.org/orexis/planning#GroundGraph> .
+  <http://example.org/orexis/graph/ground/2026-01-01T121500p0000> dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:start "2026-01-01T12:15:00+00:00"^^xsd:dateTime ] .
+  <http://example.org/orexis/graph/ground/2026-01-01T121500p0000> orexis:arrivedBy orexis:Derived .
+  <http://example.org/orexis/graph/ground/2026-01-01T121500p0000> orexis:hash "sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" .
+  <http://example.org/orexis/graph/ground/2026-01-01T121500p0000> prov:wasDerivedFrom <http://example.org/orexis/graph/ground/2026-01-01T120000p0000> .
   result:zz_moisture a orexis:BeliefGraph .
+  result:zz_moisture a orexis:Graph .
   result:zz_moisture a sensing:ResultGraph .
   result:zz_moisture dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:15:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   result:zz_moisture orexis:arrivedBy orexis:Received .
@@ -90,3 +100,12 @@
   sensed:zz_moisture orexis:arrivedBy orexis:Received .
   sensed:zz_moisture orexis:beliefsOf :keeper .
 }
+
+
+GRAPH <http://example.org/orexis/graph/ground/2026-01-01T120000p0000> {
+  orexis:obs_zz_moisture a orexis:band.zz.moisture.inside .
+  orexis:obs_zz_moisture a sensing:InRegion .
+  orexis:obs_zz_moisture a sosa:Observation .
+  orexis:obs_zz_moisture sosa:hasFeatureOfInterest :zz .
+  orexis:obs_zz_moisture sosa:observedProperty :moisture .
+}
