--- case
+++ expected
@@ -22,14 +22,19 @@
 
 GRAPH :catalogue {
   :again a orexis:Graph .
+  :again orexis:hash "sha256:d517b2a38ee8470c4604d01120d49367be1f18fa7d8338778541559cdba22ba0" .
   :catalogue a orexis:CatalogueGraph .
   :catalogue a orexis:Graph .
   :cyclic a orexis:Graph .
+  :cyclic orexis:hash "sha256:aadd0b6db809bf9a1b265af5c6e55fd39eb3c66fa4d6d8de3b3e2c9559a7acf7" .
   :deeper a orexis:Graph .
+  :deeper orexis:hash "sha256:6a4c119436bdadc75369e91fb237c380a7da33c8275e86d0ac0a708c7ff9a724" .
   :minted a orexis:Graph .
+  :minted orexis:hash "sha256:d517b2a38ee8470c4604d01120d49367be1f18fa7d8338778541559cdba22ba0" .
   :ontology a orexis:Graph .
   :ontology a orexis:OntologyGraph .
   :ontology a orexis:PublicGraph .
+  :ontology orexis:hash "sha256:da6a67b23d29f938392d8a6a3169f3fc88dc84e3ae323d95311b61e03b48ed19" .
 }
 
 
