--- case
+++ expected
@@ -29,7 +29,6 @@
 GRAPH :actions {
   :fill a orexis:Action .
   :fill orexis:takes :tank .
-  :fill planning:landsAfter """SELECT ?least ?most WHERE { BIND(60 AS ?least) BIND(180 AS ?most) }""" .
   :fill planning:costs """PREFIX : <http://example.org/test#>
       SELECT (1 AS ?cost) WHERE { }""" .
   :fill planning:effect [ a planning:Effect ; sh:rule [ a sh:SPARQLRule ; planning:update """PREFIX : <http://example.org/test#>
@@ -38,6 +37,7 @@
       CONSTRUCT { ?tank :level ?next } WHERE {
       $tank :level ?l .
       BIND($tank AS ?tank) BIND(?l + 5 AS ?next) }""" ] ] .
+  :fill planning:landsAfter "SELECT ?least ?most WHERE { BIND(60 AS ?least) BIND(180 AS ?most) }" .
   :fill planning:precondition """PREFIX : <http://example.org/test#>
       SELECT ?tank WHERE {
       ?tank a :Tank ; :level ?l .
@@ -98,6 +98,20 @@
   ground:2026-01-01T120000p0000.for.keeper.in_range planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ] .
   ground:2026-01-01T120000p0000.for.keeper.in_range planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank2 ] .
   ground:2026-01-01T120000p0000.for.keeper.in_range planning:weighs ground:2026-01-01T120000p0000 .
+  possible:1 a planning:PossibleGraph .
+  possible:1 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:03:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:01:00+00:00"^^xsd:dateTime ] .
+  possible:1 orexis:arrivedBy orexis:Derived .
+  possible:1 orexis:hash "sha256:c80ff1f6e0ffce1c8575ebd35a6183b376384efef6139df68098f31e3b94921e" .
+  possible:1 planning:by possible:fill-tank1.by .
+  possible:1 planning:minted 1 .
+  possible:1 planning:spent 1.0 .
+  possible:2 a planning:PossibleGraph .
+  possible:2 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:03:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:01:00+00:00"^^xsd:dateTime ] .
+  possible:2 orexis:arrivedBy orexis:Derived .
+  possible:2 orexis:hash "sha256:b925a49cc433542befaf03149f9d7f1e92e07473431966218677da6fae49dc78" .
+  possible:2 planning:by possible:fill-tank2.by .
+  possible:2 planning:minted 2 .
+  possible:2 planning:spent 1.0 .
   possible:fill-tank1.by :tank :tank1 .
   possible:fill-tank1.by a planning:Candidate .
   possible:fill-tank1.by planning:fills :fill .
@@ -142,3 +156,14 @@
   :keeper.in_range.pursued.tank2.met sh:property [ planning:about :level ; sh:minInclusive 10 ; sh:path :level ] .
   :keeper.in_range.pursued.tank2.met sh:targetNode :tank2 .
 }
+
+
+GRAPH possible:1 {
+  :tank1 :level 5 .
+  :tank2 :level 0 .
+}
+
+GRAPH possible:2 {
+  :tank1 :level 0 .
+  :tank2 :level 5 .
+}
