--- case
+++ expected
@@ -94,6 +94,12 @@
   ground:2026-01-01T120000p0000.for.keeper.in_range planning:met false .
   ground:2026-01-01T120000p0000.for.keeper.in_range planning:violation [ orexis:about :level ; planning:constraint 0 ; planning:instance :tank1 ] .
   ground:2026-01-01T120000p0000.for.keeper.in_range planning:weighs ground:2026-01-01T120000p0000 .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:met false .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:open true .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:violation [ orexis:about :level ; planning:constraint 0 ; planning:instance :tank1 ] .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:weighs ground:2026-01-01T120000p0000 .
   pursued:2026-01-01T120000p0000--open a orexis:Graph .
   pursued:2026-01-01T120000p0000--open a orexis:WantGraph .
   pursued:2026-01-01T120000p0000--open dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
