--- case
+++ expected
@@ -104,9 +104,9 @@
   ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ] .
   ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:weighs ground:2026-01-01T120000p0000 .
   possible:fill-tank1 a planning:PossibleGraph .
+  possible:fill-tank1 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   possible:fill-tank1 orexis:arrivedBy orexis:Derived .
   possible:fill-tank1 orexis:hash "sha256:c80ff1f6e0ffce1c8575ebd35a6183b376384efef6139df68098f31e3b94921e" .
-  possible:fill-tank1 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   possible:fill-tank1 planning:by possible:fill-tank1.by .
   possible:fill-tank1 planning:minted 1 .
   possible:fill-tank1 planning:spent 1.0 .
@@ -115,9 +115,9 @@
   possible:fill-tank1.by planning:fills :fill .
   possible:fill-tank1.by planning:from ground:2026-01-01T120000p0000 .
   possible:fill-tank1.fill-tank1 a planning:PossibleGraph .
+  possible:fill-tank1.fill-tank1 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   possible:fill-tank1.fill-tank1 orexis:arrivedBy orexis:Derived .
   possible:fill-tank1.fill-tank1 orexis:hash "sha256:262245b8ab6803ce947b3d056e7289f4f5b6c0c8a36cd94f06c2cb130be5235f" .
-  possible:fill-tank1.fill-tank1 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   possible:fill-tank1.fill-tank1 planning:by possible:fill-tank1.fill-tank1.by .
   possible:fill-tank1.fill-tank1 planning:minted 3 .
   possible:fill-tank1.fill-tank1 planning:spent 2.0 .
@@ -130,9 +130,9 @@
   possible:fill-tank1.fill-tank1.for.keeper.in_range.pursued.tank1 planning:met true .
   possible:fill-tank1.fill-tank1.for.keeper.in_range.pursued.tank1 planning:weighs possible:fill-tank1.fill-tank1 .
   possible:fill-tank1.fill-tank2 a planning:PossibleGraph .
+  possible:fill-tank1.fill-tank2 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   possible:fill-tank1.fill-tank2 orexis:arrivedBy orexis:Derived .
   possible:fill-tank1.fill-tank2 orexis:hash "sha256:886db30e45d2ce9df71bfa463e64da2ed0b025d20c419774efde962241fa9582" .
-  possible:fill-tank1.fill-tank2 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   possible:fill-tank1.fill-tank2 planning:by possible:fill-tank1.fill-tank2.by .
   possible:fill-tank1.fill-tank2 planning:minted 4 .
   possible:fill-tank1.fill-tank2 planning:spent 2.0 .
@@ -153,9 +153,9 @@
   possible:fill-tank1.for.keeper.in_range.pursued.tank1 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ] .
   possible:fill-tank1.for.keeper.in_range.pursued.tank1 planning:weighs possible:fill-tank1 .
   possible:fill-tank2 a planning:PossibleGraph .
+  possible:fill-tank2 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   possible:fill-tank2 orexis:arrivedBy orexis:Derived .
   possible:fill-tank2 orexis:hash "sha256:b925a49cc433542befaf03149f9d7f1e92e07473431966218677da6fae49dc78" .
-  possible:fill-tank2 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   possible:fill-tank2 planning:by possible:fill-tank2.by .
   possible:fill-tank2 planning:minted 2 .
   possible:fill-tank2 planning:spent 1.0 .
@@ -163,10 +163,42 @@
   possible:fill-tank2.by a planning:Candidate .
   possible:fill-tank2.by planning:fills :fill .
   possible:fill-tank2.by planning:from ground:2026-01-01T120000p0000 .
+  possible:fill-tank2.fill-tank1 a planning:PossibleGraph .
+  possible:fill-tank2.fill-tank1 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  possible:fill-tank2.fill-tank1 orexis:arrivedBy orexis:Derived .
+  possible:fill-tank2.fill-tank1 orexis:hash "sha256:886db30e45d2ce9df71bfa463e64da2ed0b025d20c419774efde962241fa9582" .
+  possible:fill-tank2.fill-tank1 planning:by possible:fill-tank2.fill-tank1.by .
+  possible:fill-tank2.fill-tank1 planning:minted 5 .
+  possible:fill-tank2.fill-tank1 planning:spent 2.0 .
+  possible:fill-tank2.fill-tank1.by :tank :tank1 .
+  possible:fill-tank2.fill-tank1.by a planning:Candidate .
+  possible:fill-tank2.fill-tank1.by planning:fills :fill .
+  possible:fill-tank2.fill-tank1.by planning:from possible:fill-tank2 .
+  possible:fill-tank2.fill-tank1.by.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  possible:fill-tank2.fill-tank1.by.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  possible:fill-tank2.fill-tank1.by.for.keeper.in_range.pursued.tank1 planning:repeats possible:fill-tank1.fill-tank2 .
+  possible:fill-tank2.fill-tank1.by.for.keeper.in_range.pursued.tank1 planning:weighs possible:fill-tank2.fill-tank1.by .
+  possible:fill-tank2.fill-tank2 a planning:PossibleGraph .
+  possible:fill-tank2.fill-tank2 dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  possible:fill-tank2.fill-tank2 orexis:arrivedBy orexis:Derived .
+  possible:fill-tank2.fill-tank2 orexis:hash "sha256:71af2c683666133f094033222ad82f45676a17913532f4ca8f587ecc30cc9993" .
+  possible:fill-tank2.fill-tank2 planning:by possible:fill-tank2.fill-tank2.by .
+  possible:fill-tank2.fill-tank2 planning:minted 6 .
+  possible:fill-tank2.fill-tank2 planning:spent 2.0 .
+  possible:fill-tank2.fill-tank2.by :tank :tank2 .
+  possible:fill-tank2.fill-tank2.by a planning:Candidate .
+  possible:fill-tank2.fill-tank2.by planning:fills :fill .
+  possible:fill-tank2.fill-tank2.by planning:from possible:fill-tank2 .
+  possible:fill-tank2.fill-tank2.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  possible:fill-tank2.fill-tank2.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
+  possible:fill-tank2.fill-tank2.for.keeper.in_range.pursued.tank1 planning:met false .
+  possible:fill-tank2.fill-tank2.for.keeper.in_range.pursued.tank1 planning:open true .
+  possible:fill-tank2.fill-tank2.for.keeper.in_range.pursued.tank1 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ] .
+  possible:fill-tank2.fill-tank2.for.keeper.in_range.pursued.tank1 planning:weighs possible:fill-tank2.fill-tank2 .
   possible:fill-tank2.for.keeper.in_range.pursued.tank1 a planning:Weighing .
+  possible:fill-tank2.for.keeper.in_range.pursued.tank1 planning:expanded true .
   possible:fill-tank2.for.keeper.in_range.pursued.tank1 planning:for :keeper.in_range.pursued.tank1 .
   possible:fill-tank2.for.keeper.in_range.pursued.tank1 planning:met false .
-  possible:fill-tank2.for.keeper.in_range.pursued.tank1 planning:open true .
   possible:fill-tank2.for.keeper.in_range.pursued.tank1 planning:violation [ planning:about :level ; planning:constraint 0 ; planning:instance :tank1 ] .
   possible:fill-tank2.for.keeper.in_range.pursued.tank1 planning:weighs possible:fill-tank2 .
   pursued:2026-01-01T120000p0000--open a orexis:Graph .
@@ -223,3 +255,14 @@
   :keeper.in_range.pursued.tank2.met sh:property [ planning:about :level ; sh:minInclusive 10 ; sh:path :level ] .
   :keeper.in_range.pursued.tank2.met sh:targetNode :tank2 .
 }
+
+
+GRAPH possible:fill-tank2.fill-tank1 {
+  :tank1 :level 5 .
+  :tank2 :level 5 .
+}
+
+GRAPH possible:fill-tank2.fill-tank2 {
+  :tank1 :level 0 .
+  :tank2 :level 10 .
+}
