--- case
+++ expected
@@ -105,9 +105,9 @@
   ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:violation [ orexis:about :level ; planning:constraint 0 ; planning:instance :tank1 ] .
   ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank1 planning:weighs ground:2026-01-01T120000p0000 .
   ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 a planning:Weighing .
+  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:expanded true .
   ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
   ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:met false .
-  ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:open true .
   ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:violation [ orexis:about :level ; planning:constraint 0 ; planning:instance :tank2 ] .
   ground:2026-01-01T120000p0000.for.keeper.in_range.pursued.tank2 planning:weighs ground:2026-01-01T120000p0000 .
   plan:keeper.in_range.pursued.tank1 a planning:PlanGraph .
@@ -161,6 +161,12 @@
   possible:fill-tank1.for.keeper.in_range.pursued.tank1 planning:met false .
   possible:fill-tank1.for.keeper.in_range.pursued.tank1 planning:violation [ orexis:about :level ; planning:constraint 0 ; planning:instance :tank1 ] .
   possible:fill-tank1.for.keeper.in_range.pursued.tank1 planning:weighs possible:fill-tank1 .
+  possible:fill-tank1.for.keeper.in_range.pursued.tank2 a planning:Weighing .
+  possible:fill-tank1.for.keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
+  possible:fill-tank1.for.keeper.in_range.pursued.tank2 planning:met false .
+  possible:fill-tank1.for.keeper.in_range.pursued.tank2 planning:open true .
+  possible:fill-tank1.for.keeper.in_range.pursued.tank2 planning:violation [ orexis:about :level ; planning:constraint 0 ; planning:instance :tank2 ] .
+  possible:fill-tank1.for.keeper.in_range.pursued.tank2 planning:weighs possible:fill-tank1 .
   possible:fill-tank2 a planning:PossibleGraph .
   possible:fill-tank2 orexis:arrivedBy orexis:Derived .
   possible:fill-tank2 orexis:hash "sha256:b925a49cc433542befaf03149f9d7f1e92e07473431966218677da6fae49dc78" .
@@ -210,6 +216,12 @@
   possible:fill-tank2.for.keeper.in_range.pursued.tank1 planning:met false .
   possible:fill-tank2.for.keeper.in_range.pursued.tank1 planning:violation [ orexis:about :level ; planning:constraint 0 ; planning:instance :tank1 ] .
   possible:fill-tank2.for.keeper.in_range.pursued.tank1 planning:weighs possible:fill-tank2 .
+  possible:fill-tank2.for.keeper.in_range.pursued.tank2 a planning:Weighing .
+  possible:fill-tank2.for.keeper.in_range.pursued.tank2 planning:for :keeper.in_range.pursued.tank2 .
+  possible:fill-tank2.for.keeper.in_range.pursued.tank2 planning:met false .
+  possible:fill-tank2.for.keeper.in_range.pursued.tank2 planning:open true .
+  possible:fill-tank2.for.keeper.in_range.pursued.tank2 planning:violation [ orexis:about :level ; planning:constraint 0 ; planning:instance :tank2 ] .
+  possible:fill-tank2.for.keeper.in_range.pursued.tank2 planning:weighs possible:fill-tank2 .
   pursued:2026-01-01T120000p0000--open a orexis:Graph .
   pursued:2026-01-01T120000p0000--open a orexis:WantGraph .
   pursued:2026-01-01T120000p0000--open dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
