--- case
+++ expected
@@ -75,9 +75,9 @@
   :ontology a orexis:Graph .
   :ontology a orexis:OntologyGraph .
   :ontology a orexis:PublicGraph .
-  :scopes a planning:ScopeGraph .
   :scopes a orexis:Graph .
   :scopes a orexis:WorkingGraph .
+  :scopes a planning:ScopeGraph .
   :scopes orexis:arrivedBy orexis:Derived .
   :scopes orexis:beliefsOf :keeper .
   :sensed a orexis:BeliefGraph .
@@ -85,8 +85,8 @@
   :sensed a orexis:StateGraph .
   :world a orexis:Graph .
   :world a orexis:PublicGraph .
+  intentions:keeper a execution:IntentionGraph .
   intentions:keeper a orexis:Graph .
-  intentions:keeper a execution:IntentionGraph .
   intentions:keeper orexis:beliefsOf :keeper .
   pursued:keeper.in_range.pursued.tank1.level a orexis:Graph .
   pursued:keeper.in_range.pursued.tank1.level a orexis:WantGraph .
