--- case
+++ expected
@@ -48,9 +48,9 @@
 GRAPH observed:probe {
   orexis:obs_probe a sosa:Observation .
   orexis:obs_probe prov:wasGeneratedBy :keeper .
-  orexis:obs_probe sensing:rawResult 0.25 .
+  orexis:obs_probe sensing:rawResult 0.08 .
   orexis:obs_probe sosa:madeBySensor :probe .
-  orexis:obs_probe sosa:resultTime "2026-01-01T11:45:00+00:00"^^xsd:dateTime .
+  orexis:obs_probe sosa:resultTime "2026-01-01T12:00:00+00:00"^^xsd:dateTime .
 }
 
 GRAPH :catalogue {
@@ -65,7 +65,7 @@
   observed:probe a orexis:Graph .
   observed:probe a orexis:StateGraph .
   observed:probe a sensing:ObservationGraph .
-  observed:probe dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:00:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T11:45:00+00:00"^^xsd:dateTime ] .
+  observed:probe dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:30:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
   observed:probe orexis:arrivedBy orexis:Received .
   observed:probe orexis:beliefsOf :keeper .
 }
