--- case
+++ expected
@@ -33,4 +33,18 @@
   :ontology a orexis:Graph .
   :ontology a orexis:OntologyGraph .
   :ontology a orexis:PublicGraph .
+  graph:scopes a deliberation:ScopeGraph .
+  graph:scopes a orexis:Graph .
+  graph:scopes a orexis:WorkingGraph .
+  graph:scopes orexis:arrivedBy orexis:Derived .
 }
+
+
+GRAPH graph:scopes {
+  :heat deliberation:inScope scopes:2 .
+  :move deliberation:inScope scopes:1 .
+  :standsIn deliberation:inScope scopes:1 .
+  scopes:1 a deliberation:Scope .
+  scopes:2 a deliberation:Scope .
+  sosa:hasSimpleResult deliberation:inScope scopes:2 .
+}
