--- case
+++ expected
@@ -28,4 +28,16 @@
   :ontology a orexis:Graph .
   :ontology a orexis:OntologyGraph .
   :ontology a orexis:PublicGraph .
+  graph:scopes a deliberation:ScopeGraph .
+  graph:scopes a orexis:Graph .
+  graph:scopes a orexis:WorkingGraph .
+  graph:scopes orexis:arrivedBy orexis:Derived .
 }
+
+
+GRAPH graph:scopes {
+  :fix deliberation:inScope scopes:1 .
+  :level deliberation:inScope scopes:1 .
+  :temperature deliberation:inScope scopes:1 .
+  scopes:1 a deliberation:Scope .
+}
