--- case
+++ expected
@@ -8,9 +8,11 @@
 
 
 GRAPH :ontology {
-  deliberation:ScopeGraph rdfs:subClassOf orexis:WorkingGraph , orexis:Graph .
+  deliberation:ScopeGraph rdfs:subClassOf orexis:Graph ;
+      rdfs:subClassOf orexis:WorkingGraph .
   orexis:WorkingGraph rdfs:subClassOf orexis:Graph .
 }
+
 
 GRAPH :actions {
   :fix a orexis:Action ;
@@ -18,8 +20,26 @@
         CONSTRUCT { ?t :level ?x . ?t :temperature ?x . } WHERE { ?t a :Tank }""" .
 }
 
+
 GRAPH :catalogue {
-  :catalogue a orexis:CatalogueGraph , orexis:Graph .
-  :ontology a orexis:OntologyGraph , orexis:PublicGraph , orexis:Graph .
-  :actions a orexis:PublicGraph , orexis:Graph .
+  :actions a orexis:Graph ;
+      a orexis:PublicGraph .
+  :catalogue a orexis:CatalogueGraph ;
+      a orexis:Graph .
+  :ontology a orexis:Graph ;
+      a orexis:OntologyGraph ;
+      a orexis:PublicGraph .
+  <http://example.org/orexis/graph/scopes> a deliberation:ScopeGraph ;
+      a orexis:Graph ;
+      a orexis:WorkingGraph ;
+      orexis:arrivedBy orexis:Derived .
 }
+
+
+
+GRAPH <http://example.org/orexis/graph/scopes> {
+  :fix deliberation:inScope scopes:1 .
+  :level deliberation:inScope scopes:1 .
+  :temperature deliberation:inScope scopes:1 .
+  scopes:1 a deliberation:Scope .
+}
