--- case
+++ expected
@@ -102,4 +102,17 @@
   :self orexis:beliefsOf :keeper .
   :world a orexis:Graph .
   :world a orexis:PublicGraph .
+  believed:bed_SoilMoisture a orexis:BeliefGraph .
+  believed:bed_SoilMoisture a orexis:Graph .
+  believed:bed_SoilMoisture a orexis:StateGraph .
+  believed:bed_SoilMoisture a sensing:SubjectBeliefGraph .
+  believed:bed_SoilMoisture dcterms:temporal [ a dcterms:PeriodOfTime ; orexis:end "2026-01-01T12:20:00+00:00"^^xsd:dateTime ; orexis:start "2026-01-01T12:00:00+00:00"^^xsd:dateTime ] .
+  believed:bed_SoilMoisture orexis:arrivedBy orexis:Derived .
+  believed:bed_SoilMoisture orexis:beliefsOf :keeper .
 }
+
+
+GRAPH believed:bed_SoilMoisture {
+  :bed :moisture 0.2999 .
+  :bed :soil :Dry .
+}
